pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/ParticipantDao.sql.stg

```diff
@@ -24,6 +24,7 @@ select en.user_study_enrollment_id as enrollment_id,
        up.sex                   as p_sex,
        up.birth_date            as p_birth_date,
        up.do_not_contact        as p_do_not_contact,
+       up.is_deceased           as p_is_deceased,
```

Comment:
Our database stores this value, so I'm pulling it out and added it to the new `UserProfile` model class so it more fully represents the data we have in database.